source/faqs.rst

```diff
@@ -373,3 +365,204 @@ The ``--no-builds`` argument completely removes the build number from the
+support older Python versions.
+
+The idea here is to specify that any time a dependency (``zlib``, in our
+running example) is used as a build dependency, it should also be automatically
+be installed as a run dependency without having to explicitly add it as such in
+the recipe. This specification is done in the ``zlib`` recipe itself (which is
+hosted by conda-forge), so in general bioconda collaborators can just add
+``zlib`` as a build dependency.
+
+Note that we don't have to specify the version of ``zlib`` in the recipe -- it
+is pinned in that ``conda_build_config.yaml`` file we share with conda-forge.
+
+In a similar fashion, the reason that we don't have to specify ``libgcc`` as
+a *run* dependency is that ``{{ compiler('c') }}`` automatically exports
+``libgcc`` as a run dependency of any package that uses the C compiler to
+build.
+
+.. _platform-nomenclature-faq:
+
+Understanding platform nomenclature
+-----------------------------------
+
+.. datechanged:: 2023-11-18
+   Added section
+
+Different CPU chips use different architecture, so programs are written
+fundamentally differently for them. A package with compiled dependencies must
+therefore use platform-specific dependencies.
+
+There is a lot of confusing nomenclature surrounding them. Here is an attempt
+at clearing them up, or at least providing enough context that you can look up
+more details on your own:
+
+**instruction set, CISC, RISC, RISC-V**: The *instruction set* is the assembly
+code commands that are possible for the chip. *CISC* is "complex instruction
+set computer" which prioritizes flexibility. *RISC* is "reduced instruction set
+computer" which prioritizing power consumption (this oversimplification, but
+that's the general idea). Instruction sets can be proprietary. ARM is a company
+that licenses a widely-used proprietary reduced instruction set. RISC-V is an
+open (non-proprietary) reduced instruction set.
+
+**ARM vs ARM RISC:** ARM is a company. They make chips (for example, the ones
+used in Raspberry Pi computers). They also license the proprietary instruction
+set (for example, they license it to Apple to run on their M-series chips).
+
+``x86_64``, ``amd64``: These are synonyms for the original Intel/AMD
+architecture.
+
+``linux/x86_64``, ``linux/arm64``, ``darwin/amd64``: These are the platform
+designators when using Docker (see `multi-platform images
+<https://docs.docker.com/build/building/multi-platform/>`_ in the Docker
+documentation).
+
+``linux-64``, ``linux-aarch64``, ``osx-64``, ``osx-arm64``: These are the
+platform designators used by conda in channels hosted by Anaconda.
+
+``aarch64``, ``arm64``: These are synonyms for ARM 64-bit architecture.
+
+**M1, M2, M3, Apple Silicon**: These are chips made by Apple and used in Macs.
+Apple licenses the ARM RISC, so they are considered aarch64 or arm64.
+
+Here is a summary table:
+
+.. list-table::
+
+  * - Linux machines from past few decades
```

Comment:
```suggestion
  * - Linux machines from past few decades (and synonyms)
```

Reply:
Can you clarify what you mean by synonyms? That the listed terms (x86_64, amd64 etc) are synonyms of each other? I guess they are for for the other table lines as well, so this could be added in the table description. Or do you mean synonyms of Linux?

Reply:
I meant that the listed terms (x86_64, amd64 etc) are synonyms of each other. Or at least partially.
Maybe I don't understand the table. But we want to say "If someone says to you - this is an linux-64 computer, it can be an x86_64 or amd64, for conda it does not matter" ... correct?

It's just nitpicking. I just thought it could be confusing if people think that those 4 terms are totally different things.

Reply:
I find your explanation above clear and this table more confusing. Does it bring any additional value?